packages/react-static/src/static/__tests__/getConfig.test.js

```diff
@@ -101,7 +101,7 @@ describe('buildConfig', () => {
     test('when no user-supplied assetsPath exists', () => {
       const { config } = buildConfig({}, { assetsPath: undefined })
 
-      expect(config.assetsPath).toBe('')
+      expect(config.assetsPath).toBeUndefined()
```

Comment:
Does this mean that we _no longer_ default to `''`? I like this/your code better btw!

Reply:
That seems to be what's happening at the moment, just that the tests were outdated. Let me give it a shot first anyways, just in case I can also fix it to be `''`

Reply:
@SleeplessByte Done! It should be correctly fixed now